ui/options_renaming.ui

```diff
@@ -222,6 +222,24 @@ p, li { white-space: pre-wrap; }
         </layout>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="ile_naming_format_documentation">
+        <property name="text">
+         <string><![CDATA[
```

Comment:
While it's possible to set the labels text here, please add the link in https://github.com/musicbrainz/picard/blob/master/picard/ui/options/renaming.py#L160 similar to how it's done in #440 to keep the two of them consistent.